### DISCUSSION TITLE
Copilot + VSCode = Infinity login loop

### DISCUSSION THREAD

**Discussion by neoacevedo** [community/community#48057] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I don't know if set this discussion in Product Feedback or in Bug but when I try to login with GitHub in VSCode I am being redirected once and a once to the OAuth login.

At the end the error message is:
> GitHub Copilot could not connect to server. Extension activation failed: "Cancelled".

Now, trying with another method suggested by the extension does the trick but another error appears:
>[INFO] [auth] [2023-02-20T15:29:53.822Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-02-20T15:29:53.823Z] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."

The second one has been already reported with "success" deployment but what I see is that it still has the issue.

**Comment by Akash1134** [comment]
Hi @neoacevedo ,

This error indicates that there was an error connecting to the GitHub API to request a token to use GitHub Copilot, or you do not have a GitHub Copilot subscription.

To request another token from api.github.com, try signing in and out of GitHub Copilot from your IDE. Once you've logged out, GitHub Copilot will prompt you to sign back in.

If that does not help, please check out this [discussion](https://github.com/orgs/community/discussions/47577).
